tkltest/util/unit/coverage_util.py

```diff
@@ -238,32 +258,29 @@ def get_delta_coverage(test, test_raw_cov_file, ctd_raw_cov_file, main_coverage_
             # If merging failed we skip current test file and assume it resulted in zero delta coverage
             # The reason to continue is that we may still gain from previous augmenting test files
             tkltest_status('Warning: merging of jacoco output failed, skipping current test file: {}\n{}'.format(e, e.stderr))
-            return {
-                       'instruction_cov_delta': 0,
-                       'line_cov_delta': 0,
-                       'branch_cov_delta': 0,
-                       'method_cov_delta': 0
-                   }, {'instruction_covered': base_coverage['instruction_covered'],
-                       'line_covered': base_coverage['line_covered'],
-                       'branch_covered': base_coverage['branch_covered'],
-                       'method_covered': base_coverage['method_covered'],
-                       'instruction_total': base_coverage['instruction_total'],
-                       'line_total': base_coverage['line_total'],
-                       'branch_total': base_coverage['branch_total'],
-                       'method_total': base_coverage['method_total']}
-    else:
+            return no_delta_coverage
+    elif os.path.isfile(test_raw_cov_file):
         shutil.copy(test_raw_cov_file, output_exec_file)
+    else:
+        tkltest_status('Warning: file {} does not exist'.format(test_raw_cov_file))
```

Comment:
indicate what this file is (not only its name): e.g.,  coverage data file for test <test-name>